src/app/app.module.ts

```diff
@@ -26,6 +26,10 @@ import {DiscordEmbedModule} from './shared/discord-embed/discord-embed.module';
 import {HomebrewSharingModule} from './homebrew-sharing/homebrew-sharing.module';
 import {LoginModule} from './login/login.module';
 
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({ dsn: 'https://af2b06560981446bb55f64b6f79fd520@sentry.io/1486249' });
```

Comment:
We might want to consider moving the error utils to a package so we can have the helper methods that Jason wrote back for the twitch extension. We've made a few minor fixes in the copy the encounter builder has.

https://github.com/DnDBeyond/ddb-encounter-builder-client/blob/master/src/error/ErrorUtils.js

Reply:
I will log this.

Reply:
See: #26